packages/cli/src/codemods/v0-8/ui-package-split.ts

```
@@ -23,49 +23,181 @@ const reactUIExports = [
+      return (
+        arg &&
+        (arg.type === "Literal" || arg.type === "StringLiteral") &&
+        arg.value === "@assistant-ui/react-markdown/tailwindcss"
+      );
+    })
+    .forEach((path: any) => {
+      removedMarkdownPlugin = true;
+      const parent = path.parentPath;
+      if (
+        parent &&
+        parent.value &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)